pta-intelligent-search-metadata-extractor/src/main/java/fi/maanmittauslaitos/pta/search/metadata/ISOMetadataExtractorConfigurationFactory.java

```diff
@@ -143,52 +145,227 @@ public DocumentProcessingConfiguration createMetadataDocumentProcessingConfigura
+				ResultMetadataFields.CRS_VERSION,
+				FieldExtractorType.FIRST_MATCHING_VALUE,
+				"(" +
+						XPATH_CRS_ROOT +
+						"/*/gmd:version/gco:CharacterString/text())"));
+		// Scale data
+		extractors.add(createXPathExtractor(
+				ResultMetadataFields.SCALE_DENOMINATOR,
+				FieldExtractorType.FIRST_MATCHING_VALUE,
+				"(//gmd:identificationInfo/*/gmd:spatialResolution/*/gmd:equivalentScale/*/gmd:denominator/gco:Integer/text())"));
+
+		// Extent
+		// North / Top
+		final String XPATH_EXTENT_ROOT = "//gmd:identificationInfo/*/*[self::gmd:extent or self::srv:extent]/*/gmd:geographicElement";
+
+
 		// Geographic bounding box
 		extractors.add(createXPathExtractor(
 				ResultMetadataFields.GEOGRAPHIC_BOUNDING_BOX,
 				new GeographicBoundingBoxXmlCustomExtractor(),
 				"//gmd:MD_Metadata/gmd:identificationInfo/*/*[self::gmd:extent or self::srv:extent]/*/gmd:geographicElement/gmd:EX_GeographicBoundingBox"));
 
+		extractors.add(createXPathExtractor(
+				ResultMetadataFields.EXTENT_NORTH_BOUND,
+				FieldExtractorType.FIRST_MATCHING_VALUE,
+				"(" +
+						XPATH_EXTENT_ROOT +
+						"/*/gmd:northBoundLatitude/gco:Decimal/text())"));
+
+		// South / bottom
+		extractors.add(createXPathExtractor(
+				ResultMetadataFields.EXTENT_SOUTH_BOUND,
+				FieldExtractorType.FIRST_MATCHING_VALUE,
+				"(" +
+						XPATH_EXTENT_ROOT +
+						"/*/gmd:southBoundLatitude/gco:Decimal/text())"));
+
+		// East / right
+		extractors.add(createXPathExtractor(
+				ResultMetadataFields.EXTENT_EAST_BOUND,
+				FieldExtractorType.FIRST_MATCHING_VALUE,
+				"(" +
+						XPATH_EXTENT_ROOT +
+						"/*/gmd:eastBoundLongitude/gco:Decimal/text())"));
+
+		// West / left
+		extractors.add(createXPathExtractor(
+				ResultMetadataFields.EXTENT_WEST_BOUND,
+				FieldExtractorType.FIRST_MATCHING_VALUE,
+				"(" +
+						XPATH_EXTENT_ROOT +
+						"/*/gmd:westBoundLongitude/gco:Decimal/text())"));
+
+
+		// Classification
+		// List of possible values found at http://standards.iso.org/ittf/PubliclyAvailableStandards/ISO_19139_Schemas/resources/codelist/gmxCodelists.xml#MD_ClassificationCode
+		extractors.add(createXPathExtractor(
+				ResultMetadataFields.CLASSIFICATION,
+				FieldExtractorType.FIRST_MATCHING_VALUE,
+				"(//gmd:identificationInfo/*/gmd:resourceConstraints/*/gmd:classification/*/@codeListValue)"));
+
+		extractors.add(createXPathExtractor(
+				ResultMetadataFields.DOWNLOAD_LINKS,
+				new DownloadLinksXmlCustomExtractor(),
+				"(//gmd:distributionInfo/*/gmd:transferOptions/*/gmd:onLine)"));
+
+		extractors.add(createXPathExtractor(
+				ResultMetadataFields.SERVICE_ASSOCIATED_RESOURCES,
+				new ServiceAssociatedResourcesXmlCustomExtractor(),
+				"(//gmd:identificationInfo/*/srv:operatesOn)"));
+
+
+		// Additional extractors for other documents
+		// A resources additional resources need to be queried in a separate query, so its included under a separate topic/group
+		// Associated resources
+		extractors.add(createXPathExtractor(
```

Comment:
This does not belong here. This configuration should only be for extracting information from MD_Metadata resources. Please move out of the pull request and do this customisation at the "alusta"

Reply:
This has been moved back to PTA